ui/components/ui/identicon/blockieIdenticon/blockieIdenticon.component.js

```diff
@@ -19,7 +19,17 @@ const BlockieIdenticon = ({ address, diameter, alt }) => {
   return (
     <>
       <canvas ref={canvasRef} style={{ display: 'none' }} />
-      <img src={dataUrl} height={diameter} width={diameter} alt={alt || ''} />
+      <img
+        src={dataUrl}
+        height={diameter}
+        width={diameter}
+        style={
+          {
+            borderRadius: style,
```

Comment:
` || null` will never be reached here because we're defining an object first.  Also, since `style` only represents what will become a `borderRadius`, maybe we should name that param `borderRadius`?

Reply:
done